[2025-10-01T15:16:30.755+0000] {scheduler_job_runner.py:769} ERROR - Executor reports task instance <TaskInstance: main.run_pyspark_task manual__2025-10-01T15:16:28.890598+00:00 [queued]> finished (failed) although the task says it's queued. (Info: None) Was the task killed externally?
